First draft of CLI import docs (#1276)
checkly/docs.checklyhq.com: site/content/docs/cli/import.md

---
title: Importing your account - Checkly Docs

> displayTitle: Importing your account to the Checkly CLI
> navTitle: Importing an existing account

Reviewer: In accordance to our discussion on the UI nudges, I would change this to:

```suggestion
displayTitle: Importing resources with the Checkly CLI
navTitle: Importing existing resources```
